site/content/docs/4.3/components/navbar.md

```diff
@@ -66,9 +66,13 @@ Here's an example of all the sub-components included in a responsive light-theme
           <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>
         </li>
       </ul>
-      <form class="form-inline my-2 my-lg-0">
-        <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
-        <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+      <form class="row g-2 mt-3 mt-lg-0 pl-lg-2">
+        <div class="col-auto">
```

Comment:
Know we're saving on CSS with using the grid for inline forms, but I imagine folks will complain about this extra HTML. Just calling out for awareness, don't make any changes for this just yet.

Reply:
Indeed, the grid is a bit overkill to align these. I've used `.d-flex` here.